backend/src/journal/dto/update-journal.input.ts

```diff
@@ -0,0 +1,13 @@
+import { InputType, Field } from '@nestjs/graphql';
+
+@InputType()
+export class UpdateJournalInput {
+  @Field({ nullable: true })
+  id?: string;
```

Comment:
You don't need an ID here

In your mutation you pass ID as a separate property, while this UpdateJournalInput only contains fields that can be updated

Reply:
Done!